packages/cli-platform-android/src/tools/filterGradlewOutput.ts

```diff
@@ -0,0 +1,26 @@
+import {logger} from '@react-native-community/cli-tools';
+
+/**
+ *
+ * @param output
+
+  This function takes the output of the gradlew command and filters out the deprecation warning about setting the namespace in `AndroidManifest.xml`.
+
+  Warning:
+  package="com.library" found in source AndroidManifest.xml: /Users/user/app/node_modules/library/android/src/main/AndroidManifest.xml.
+  Setting the namespace via a source AndroidManifest.xml's package attribute is deprecated.
+
+  Please instead set the namespace (or testNamespace) in the module's build.gradle file, as described here: https://developer.android.com/studio/build/configure-app-module#set-namespace
+
+  This migration can be done automatically using the AGP Upgrade Assistant, please refer to https://developer.android.com/studio/build/agp-upgrade-assistant for more information.
+ */
+
+export const filterGradlewOutput = (output: string) => {
+  const regex = new RegExp(
+    /package="([^"]+)" found in source AndroidManifest\.xml:(.*?)Setting the namespace via a source AndroidManifest.xml's package attribute is deprecated\.(.*?)Please instead set the namespace \(or testNamespace\) in the module's build\.gradle file, as described here: https:\/\/developer\.android\.com\/studio\/build\/configure-app-module#set-namespace(.*?\n)*This migration can be done automatically using the AGP Upgrade Assistant, please refer to https:\/\/developer\.android\.com\/studio\/build\/agp-upgrade-assistant for more information\./gms,
```

Comment:
This is a single line, we can match only a part of it. The wording may change between versions—not sure if it happened ever though

Reply:
Hm, I'm not sure what do you mean, because if we only check the first sentence "package="com.swmansion.gesturehandler.react" found in source AndroidManifest.xml:", how would we replace other part of this warning?